doc/args_nonci.rst

```diff
@@ -0,0 +1,21 @@
+:orphan:
+
+Non-CI subcommand
+-------------------
+
+Purpose of this mode is running 'Universum' configurations locally.
+An aim of this mode is using Universum as part of build system.
```

Comment:
Word 'aim' cannot be used in this context. Please rephrase, e.g. "This mode was designed to use Universum as a part..."